src/Uno.UI/AssemblyInfo.cs

```diff
@@ -5,6 +5,7 @@
 [assembly: InternalsVisibleTo("Uno.UI.Tests.Performance")]
 [assembly: InternalsVisibleTo("Uno.UI.Toolkit")]
 [assembly: InternalsVisibleTo("Uno.UI.RemoteControl")]
+[assembly: InternalsVisibleTo("Uno.UI.Runtime.Wasm")]
```

Comment:
This will need to be renamed to `Uno.UI.Runtime.WebAssembly` after #3595

Reply:
I expect to merge it before ;-) ;-) ;-)